src/inverse-client.ts

```diff
@@ -386,7 +267,7 @@ export class RestClient {
     sl_trigger_by?: string;
     new_trailing_active?: number;
   }): GenericAPIResponse {
-    return this.requestWrapper.post('open-api/position/trading-stop', params);
+    return this.requestWrapper.post('v2/private/position/trading-stop', params);
   }
 
   setUserLeverage(params: {
```

Comment:
Not sure how long it was broken, but seems this method has the wrong endpoint:
https://bybit-exchange.github.io/docs/inverse/#t-setleverage

Can you change the endpoint to `v2/private/position/leverage/save`?